### PR TITLE
[SidePanel API] Add sidePanel.open() use case

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #https://github.com/GoogleChromeLabs/Extension-Docs/issues/143

Changes proposed in this pull request:

- Add new use case
- Link to sample
- Update the way a user can open the side panel.

### CODE REVIEW

File: site/en/docs/extensions/reference/sidePanel/index.md [hunk 1]
```diff
@@ -142,6 +142,37 @@ chrome.sidePanel
...
+  if (info.menuItemId === 'openSidePanel') {
+    // This will open the panel in all the pages on the current window.
+    chrome.sidePanel.open({ windowId: tab.windowId });
+  }
+});
+```
+
+When using [`sidePanel.open()`][sidepanel-open], you must choose the context in which it should open. Use [`windowId`][sidepanel-windowid] to open a global side panel in the specified window. Alternatively, set the [`tabId`][sidepanel-tabid] to open the side panel only on a specific tab.
+
+{% Aside 'important' %}
```
Comment (AmySteam) [inline site/en/docs/extensions/reference/sidePanel/index.md:172]: @sebastianbenz I'm debating if this would fit better in the ## Overview section 🧐

Comment (sebastianbenz) [reply site/en/docs/extensions/reference/sidePanel/index.md:172]: You mean the previous or the following paragraph? I think it makes sense here as this is particularly relevant for sidepanel.open.

File: site/en/docs/extensions/reference/sidePanel/index.md [hunk 2]
```diff
@@ -182,34 +213,38 @@ Navigating to the side panel menu
   </figcaption>
 </figure>
 
-Using the action toolbar icon
-: Users can open the side panel by clicking on [the action icon](#open-action-icon) if it's enabled.
-
-Using a keyboard shortcut
```
Comment (sebastianbenz) [inline site/en/docs/extensions/reference/sidePanel/index.md:188]: Did you intentionally drop this paragraph?

File: site/en/docs/extensions/reference/sidePanel/index.md [hunk 3]
```diff
@@ -182,34 +213,38 @@ Navigating to the side panel menu
...
-: Users can open the side panel by pressing a keyboard shortcut if the [action command][action-commands] and the [action icon](#open-action-icon) are enabled.
-
+Open through a user gesture
```
Comment (sebastianbenz) [inline site/en/docs/extensions/reference/sidePanel/index.md:216]: I would explicitly list all three options here:

1. Action button
2.  Keyboard
3. Programmatically after user interaction

File: site/en/docs/extensions/reference/sidePanel/index.md (hunk 2 above)
Comment (AmySteam) [reply site/en/docs/extensions/reference/sidePanel/index.md:188]: Yeah I felt like it fell under "user gesture". I'll explicitly list it under **Open through a user gesture**

File: site/en/docs/extensions/reference/sidePanel/index.md (hunk 1 above)
Comment (AmySteam) [reply site/en/docs/extensions/reference/sidePanel/index.md:172]: I meant the aside. I'll leave it where it's al

**Comment by AmySteam** [comment]
Thanks for the feedback! @sebastianbenz PTAL =)

**Review by sebastianbenz — APPROVED** [review]
Great!

**Comment by AmySteam** [comment]
@jpmedley This is ready for a copy edit. This PR will be merged on Thu July 20th. Thanks!

File: site/en/docs/extensions/reference/sidePanel/index.md [hunk 4]
```diff
@@ -142,6 +142,37 @@ chrome.sidePanel
 With `openPanelOnActionClick()` enabled, you can also open the side panel using a keyboard shortcut by specifying an [action command][action-commands] in the manifest as featured in the [Tab-specific side panel][sample-sp-google] sample.
 {% endAside %}
 
+### Programmatically open the side panel on user interaction {: #user-interaction } 
+
+Chrome 116 introduces [`sidePanel.open()`][sidepanel-open]. It allows extensions to invoke the side panel through a user interaction, such as an [action click][api-action], a [keyboard shortcut][api-commands], a [context menu][api-menu], or a button click on an extension page or [content script][doc-cs]. For a complete demo, see the [Open Side Panel][sample-sp-open] sample extension.
```
Comment (AmySteam) [inline site/en/docs/extensions/reference/sidePanel/index.md:147]:
```suggestion
Chrome 116 introduces [`sidePanel.open()`][sidepanel-open]. It allows extensions to invoke the side panel through a user interaction, such as an [action toolbar icon click][api-action], a [keyboard shortcut][api-commands], a [context menu][api-menu], or a button click on an extension page or [content script][doc-cs]. For a complete demo, see the [Open Side Panel][sample-sp-open] sample extension.
```

WDYT?

File: site/en/docs/extensions/reference/sidePanel/index.md [hunk 5]
```diff
@@ -142,6 +142,37 @@ chrome.sidePanel
...
+  }
+});
+```
+
+When using [`sidePanel.open()`][sidepanel-open], you must choose the context in which it should open. Use [`windowId`][sidepanel-windowid] to open a global side panel in the specified window. Alternatively, set the [`tabId`][sidepanel-tabid] to open the side panel only on a specific tab.
+
+{% Aside 'important' %}
+Remember to design your side panel as a useful companion tool for users, improving their browsing experience without unnecessary distractions. Check the [Quality Guidelines][cws-quality] in the Program Policies for more info.
+{% endAside %}
+
```
[the remark this answers is not in the document]
Comment (AmySteam) [reply site/en/docs/extensions/reference/sidePanel/index.md:175]: I can move the first paragraph, but I want to leave the tip after the implementation details.

File: site/en/docs/extensions/reference/sidePanel/index.md [hunk 6]
```diff
@@ -182,34 +213,43 @@ Navigating to the side panel menu
...
-
-Using a keyboard shortcut
-: Users can open the side panel by pressing a keyboard shortcut if the [action command][action-commands] and the [action icon](#open-action-icon) are enabled.
+Open through a user gesture
+: Users can open the side panel through the user interactions using [`sidePanel.open()`](#user-interaction) and [`sidePanel.setPanelBehavior()`](#open-action-icon), such as:
 
+  - An [action click][api-action]
+  - A [keyboard shortcut][api-commands]
+  - A [context menu][api-menu]
+  - Programmatically after a [user interaction](#user-interaction).
```
[the remark this answers is not in the document]
Comment (AmySteam) [reply site/en/docs/extensions/reference/sidePanel/index.md:222]:
```suggestion
  - A [user gesture](#user-interaction) on an extension page or content script.
```

[the remark this answers is not in the document]
Comment (AmySteam) [reply site/en/docs/extensions/reference/sidePanel/index.md:222]: This is more explicit

**Review by jpmedley — APPROVED** [review]

**Comment by AmySteam** [comment]
@jpmedley Thanks for approving. I rewrote it a bit based on your input. PTAL, I think it's more explicit and clear.

This will be merged with the 116 PR's tomorrow.

**Pull request merged**